### ISSUE TITLE
Answer being Marked Incorrect when it is Correct and fits in assigned range

### ISSUE THREAD

**Issue by sandyraysor** [Simon-Initiative/oli-torus#4381]
https://proton.oli.cmu.edu/authoring/project/real_chemistry_i_taylors_versi/resource/vwx8a_the_bohr_model_of_the_atom_qp11v

BohrCalc5to2DIGT (last DIGT on page)
Range set for 1st numeric 
![image](https://github.com/Simon-Initiative/oli-torus/assets/36045601/4a0e7d8f-d31e-4e9b-8534-b89f592634bb)
Range set for 2nd numeric
![image](https://github.com/Simon-Initiative/oli-torus/assets/36045601/729b05fa-f1b1-46f7-ba26-cb6b326f1321)
Answers in student view: https://proton.oli.cmu.edu/sections/real_chemistry_1/page/vwx8a_the_bohr_model_of_the_atom_qp11v
![image](https://github.com/Simon-Initiative/oli-torus/assets/36045601/4211c581-ca6d-4667-a9f7-ff14b244c1e4)

Answers are within the set range, but it is coming up as incorrect.

**Comment by andersweinstein** [comment]
I noticed the lower bound (first number in range) is numerically greater than the upper bound (second) in both of these cases, so I experimented in the authoring page with swapping them. That worked in preview so I have left this change in place (unpublished).

I did run into some difficulty accomplishing this swap: many attempts found torus restoring the order on reload of the answer tab, as is done in switching focus to another input and then back. I think it worked to change one to some garbage, wait for save, change the second, wait for save, then fix the first. Found this a bit odd; maybe should be looked into.
